### PR TITLE
COMMERCE-9551 Use correct className for updating expando

### PR DESCRIPTION
**Pull request by brianikim**
@andrea-ale-sbarra ,
https://issues.liferay.com/browse/COMMERCE-9551

### CODE REVIEW

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:test:sf

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-account-impl/src/main/java/com/liferay/headless/commerce/admin/account/internal/resource/v1_0/AccountGroupResourceImpl.java [hunk 1]
```diff
@@ -222,7 +222,8 @@ public Response patchAccountGroupByExternalReferenceCode(
 
 		if ((customFields != null) && !customFields.isEmpty()) {
 			ExpandoUtil.updateExpando(
-				contextCompany.getCompanyId(), CommerceAccountGroup.class,
+				contextCompany.getCompanyId(),
+				com.liferay.account.model.AccountGroup.class,
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-account-impl/src/main/java/com/liferay/headless/commerce/admin/account/internal/resource/v1_0/AccountGroupResourceImpl.java:226]: Why do you need to write the whole package and not use just the class name?
`AccountGroup.class`

Comment (luca-pellizzon) [reply modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-account-impl/src/main/java/com/liferay/headless/commerce/admin/account/internal/resource/v1_0/AccountGroupResourceImpl.java:226]: AccountGroup is the DTO used in the headless, so we have to use the whole package to make sure we are using the proper class

**Review by luca-pellizzon — APPROVED** [review]

**Pull request closed without merging**